### ISSUE TITLE
Crash when retrieving `description` of Agree field via GQL

### ISSUE THREAD

**Issue by tremby** [verbb/formie#870]
**Description**

I'm seeing a new crash when trying to retrieve the `description` of an "agree"-type field.

**Steps to reproduce**
1. Make a field of type "agree". Fill in a field name and a description since they're required, everything else can be left default.
2. Query for its `description` field via GQL.
3. See error.

```
{
  "errors": [
    {
      "debugMessage": "Undefined variable $propertyName",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "file": "/srv/my-project/vendor/verbb/formie/src/base/FormFieldTrait.php",
      "line": 1033,
      "trace": [
        {
          "file": "/srv/my-project/vendor/craftcms/cms/src/web/ErrorHandler.php",
          "line": 77,
          "call": "yii\\base\\ErrorHandler::handleError(2, 'Undefined variable $propertyName', '/srv/my-project/vendor/verbb/formie/src/base/FormFieldTrait.php', 1033)"
        },
        {
          "file": "/srv/my-project/vendor/verbb/formie/src/base/FormFieldTrait.php",
          "line": 1033,
          "call": "craft\\web\\ErrorHandler::handleError(2, 'Undefined variable $propertyName', '/srv/my-project/vendor/verbb/formie/src/base/FormFieldTrait.php', 1033)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 623,
          "call": "verbb\\formie\\base\\FormField::verbb\\formie\\base\\{closure}(instance of verbb\\formie\\fields\\formfields\\Agree, array(0), array(2), instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 550,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveFieldValueOrError(instance of GraphQL\\Type\\Definition\\FieldDefinition, instance of GraphQL\\Language\\AST\\FieldNode, instance of Closure, instance of verbb\\formie\\fields\\formfields\\Agree, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1195,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Field_Agree, instance of verbb\\formie\\fields\\formfields\\Agree, instance of ArrayObject(1), array(4))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1145,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Field_Agree, instance of verbb\\formie\\fields\\formfields\\Agree, array(3), instance of ArrayObject(1))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1106,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields(GraphQLType: Field_Agree, instance of ArrayObject(1), array(3), instance of verbb\\formie\\fields\\formfields\\Agree)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 974,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue(GraphQLType: Field_Agree, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), instance of verbb\\formie\\fields\\formfields\\Agree)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 789,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: FieldInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), instance of verbb\\formie\\fields\\formfields\\Agree)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 654,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: FieldInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), instance of verbb\\formie\\fields\\formfields\\Agree)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 887,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: FieldInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(3), instance of verbb\\formie\\fields\\formfields\\Agree)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 761,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeListValue(GraphQLType: FieldInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(1))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 654,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: FieldInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(1))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: FieldInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), array(1))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1195,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Form, instance of verbb\\formie\\elements\\Form, instance of ArrayObject(1), array(2))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1145,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Form, instance of verbb\\formie\\elements\\Form, array(1), instance of ArrayObject(1))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1106,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields(GraphQLType: Form, instance of ArrayObject(1), array(1), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 974,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue(GraphQLType: Form, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 789,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 654,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: FormInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of verbb\\formie\\elements\\Form)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1195,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Query, null, instance of ArrayObject(1), array(1))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 264,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Query, null, array(0), instance of ArrayObject(1))"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 215,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeOperation(instance of GraphQL\\Language\\AST\\OperationDefinitionNode, null)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/Executor/Executor.php",
          "line": 156,
          "call": "GraphQL\\Executor\\ReferenceExecutor::doExecute()"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 162,
          "call": "GraphQL\\Executor\\Executor::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, instance of GraphQL\\Language\\AST\\DocumentNode, null, array(2), null, 'MyQuery', null)"
        },
        {
          "file": "/srv/my-project/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 94,
          "call": "GraphQL\\GraphQL::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, 'query MyQuery {\n  formieForm(handle: \"bugTest\") {\n    formFields {\n      ... on Field_Agree {\n        description\n      }\n    }\n  }\n}\n', null, array(2), null, 'MyQuery', null, array(26))"
        },
        {
          "file": "/srv/my-project/vendor/craftcms/cms/src/services/Gql.php",
          "line": 507,
          "call": "GraphQL\\GraphQL::executeQuery(instance of GraphQL\\Type\\Schema, 'query MyQuery {\n  formieForm(handle: \"bugTest\") {\n    formFields {\n      ... on Field_Agree {\n        description\n      }\n    }\n  }\n}\n', null, array(2), null, 'MyQuery', null, array(26))"
        },
        {
          "file": "/srv/my-project/vendor/craftcms/cms/src/controllers/GraphqlController.php",
          "line": 177,
          "call": "craft\\services\\Gql::executeQuery(instance of craft\\models\\GqlSchema, 'query MyQuery {\n  formieForm(handle: \"bugTest\") {\n    formFields {\n      ... on Field_Agree {\n        description\n      }\n    }\n  }\n}\n', null, 'MyQuery', true)"
        },
        {
          "call": "craft\\controllers\\GraphqlController::actionApi()"
        },
        {
          "file": "/srv/my-project/vendor/yiisoft/yii2/base/InlineAction.php",
          "line": 57,
          "function": "call_user_func_array(array(2), array(0))"
        },
        {
          "file": "/srv/my-project/vendor/yiisoft/yii2/base/Controller.php",
          "line": 178,
          "call": "yii\\base\\InlineAction::runWithParams(array(2))"
        },
        {
          "file": "/srv/my-project/vendor/yiisoft/yii2/base/Module.php",
          "line": 552,
          "call": "yii\\base\\Controller::runAction('api', array(2))"
        },
        {
          "file": "/srv/my-project/vendor/craftcms/cms/src/web/Application.php",
          "line": 301,
          "call": "yii\\base\\Module::runAction('graphql/api', array(2))"
        },
        {
          "file": "/srv/my-project/vendor/craftcms/cms/src/web/Application.php",
          "line": 625,
          "call": "craft\\web\\Application::runAction('graphql/api', array(2))"
        },
        {
          "file": "/srv/my-project/vendor/craftcms/cms/src/web/Application.php",
          "line": 280,
          "call": "craft\\web\\Application::_processActionRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/srv/my-project/vendor/yiisoft/yii2/base/Application.php",
          "line": 384,
          "call": "craft\\web\\Application::handleRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/srv/my-project/web/index.php",
          "line": 12,
          "call": "yii\\base\\Application::run()"
        }
      ]
    }
  ],
  "data": {
    "formieForm": {
      "formFields": [
        {
          "description": null
        }
      ]
    }
  }
}
```

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: n/a
- Client-side Validation: n/a
- Custom Form Templates: n/a

**Additional info**
- Plugin version: craft-4 at 4792400
- Craft version: 4.0.0.1
- Multi-site: Yes

**Additional context**

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.6](https://github.com/verbb/formie/releases/tag/2.0.0-beta.6)

**Comment by tremby** [comment]
Thanks for the quick fix!
